saleor/graphql/schema.graphql

```diff
@@ -299,6 +299,8 @@ type AppExtension implements Node {
   type: AppExtensionTypeEnum!
   target: AppExtensionTargetEnum!
   permissions: [Permission!]!
+  accessToken: String
+  canBeUsedByUser: Boolean
```

Comment:
`canBeUsedByUser: Boolean!`

Reply:
My bad, `canBeUsedByUser` will be added in separate PR as I have to (probably) add some performance improvments for these fields. Will revert this field, but thanks for catch